### PR TITLE
perf(ivy): chain multiple `i18nExp` calls

### PR DESCRIPTION
**Pull request by ocombe**
Implement function chaining for `i18nExp` to reduce the output size.

FW-1391 #resolve

### CODE REVIEW

File: packages/compiler/src/render3/view/template.ts [hunk 1]
```diff
@@ -654,13 +659,17 @@ export class TemplateDefinitionBuilder implements t.Visitor<void>, LocalResolver
...
-                    () => [this.convertExpressionBinding(expression)]);
+                bindings.push({
+                  sourceSpan: element.sourceSpan,
+                  value: () => this.convertExpressionBinding(expression)
+                });
               });
             }
           }
         });
+        if(bindings.length) {
```
Comment (AndrewKushnir) [inline packages/compiler/src/render3/view/template.ts]: Missing whitespace in `if(`.

File: packages/compiler/src/render3/view/template.ts [hunk 2]
```diff
@@ -1093,12 +1090,17 @@ export class TemplateDefinitionBuilder implements t.Visitor<void>, LocalResolver
...
-    const span = bindings.length ? bindings[0].input.sourceSpan : null;
+    const span = bindings.length ? bindings[0].sourceSpan : null;
 
     this.addSelectInstructionIfNecessary(nodeIndex, span);
     this._updateCodeFns.push(() => {
-      const calls = bindings.map(
-          property => [o.literal(property.name), property.value(), ...(property.params || [])]);
+      const calls = bindings.map(property => {
+        const fnParams = [property.value(), ...(property.params || [])];
+        if(property.name) {
```
Comment (AndrewKushnir) [inline packages/compiler/src/render3/view/template.ts]: Missing whitespace in if(.

**Review by AndrewKushnir — APPROVED** [review]
LGTM (with a couple nits) 👍

**Review by alxhub — APPROVED** [review]

**Comment by AndrewKushnir** [comment]
- [VE Presubmit](http://test/OCL:254989039:BASE:254989062:1561481378003:c7a7e629)
- [Ivy Presubmit](http://test/OCL:254989039:BASE:254989332:1561481454910:83738456)

**Review by kara — APPROVED** [review]
LGTM

**Pull request closed without merging**